### PR TITLE
add CoverageChecker

### PR DESCRIPTION
**Pull request by exussum12**
CoverageChecker allows QA tools to run on a single pull request.
This allows running these tools on a large legacy code base without having to fix all of the outstanding issues. Only the code you change needs to conform to the standard selected

Eg PSR-2 check only on changed code, or ensure every pull request has a test covering the new code

### CODE REVIEW

File: tools.json [hunk 1]
```diff
@@ -576,6 +576,18 @@
...
       },
       "test": "testability --help"
+    },
+    {
+      "name": "CoverageChecker",
+      "summary": "Applies QA tools to run on a single pull request",
+      "website": "https://github.com/exussum12/coverageChecker",
+      "command": {
+        "composer-bin-plugin": {
+          "package": "exussum12/coverage-checker:dev-master",
```
Comment (jakzal) [inline tools.json]: Can we use `exussum12/coverage-checker` here so that the latest stable version is installed?

**Comment by jakzal** [comment]
Looks interesting! Thank you for your contribution :)

File: tools.json (hunk 1 above)
Comment (exussum12) [reply tools.json]: Changed, I just copied the one from above so assumed that was the way to do it

Comment (jakzal) [reply tools.json]: Have you pushed? I can't see your changes :)

Comment (exussum12) [reply tools.json]: odd - I used the web editor. Will just use regular git

Comment (exussum12) [reply tools.json]: https://github.com/jakzal/phpqa/compare/master...exussum12:patch-1  It does appear do wont let me push. I will take a look tonight. Thanks

**Pull request closed without merging**